src/Compilers/Core/Portable/Serialization/ObjectWriter.cs

```diff
@@ -776,7 +778,7 @@ private void WritePrimitiveType(Type type, EncodingKind kind)
         public void WriteType(Type type)
         {
             _writer.Write((byte)EncodingKind.Type);
-            this.WriteString(type.AssemblyQualifiedName);
+            this.WriteString(type.AssemblyQualifiedName!);
```

Comment:
>! [](start = 55, length = 1)

Is `!` necessary? It looks like `WriteString` takes `string?`.

Reply:
Not necessary. Fixed